conjur/resource.py

```diff
@@ -28,3 +32,74 @@ def __init__(self, api, kind, identifier):
     @property
     def resourceid(self):
         return ":".join([self.api.config.account, self.kind, self.identifier])
+
+    def permit(self, role, privilege, grant_option=False):
+        '''
+        Permit `role` to perform `privilege` on this resource.  If
+        `grant_option` is True, the role will be able to grant this
+        permission to other resources.
+
+        You must own the resource or have the permission with `grant_option`
+        to call this method.
+        '''
+        data = {}
+        params = {
+            'permit': 'true',
+            'privilege': privilege,
+            'role': authzid(role, 'role')
+        }
+        if grant_option:
+            data['grant_option'] = 'true'
+
+        self.api.post(self.url(), data=data, params=params)
+
+    def deny(self, role, privilege):
+        '''
+        Deny `role` permission to perform `privilege` on this resource.
+
+        You must own the resource or have the permission with `grant_option`
+        on it to call this method.
+        '''
+        params = {
+            'permit': 'true',
+            'privilege': privilege,
+            'role': authzid(role)
+        }
+
+        self.api.post(self.url(), parmas=params)
+
+    def check_permission(self, privilege, role=None):
```

Comment:
Aliasing is super unpythonic (but very popular in ruby, even `Enumerable` and `Array` have multiple aliases for their methods).  `permitted` seems good, and worth doing before anybody starts writing code against this.